Composer/packages/client/src/recoilModel/utils/index.ts

```diff
@@ -7,6 +7,53 @@ import merge from 'lodash/merge';
 import storage from '../../utils/storage';
 import { isElectron } from '../../utils/electronUtil';
 
+const getOS = () => {
```

Comment:
I suggest moving all of these to a file called ```fontUtils.ts```, then export a function called ```getDefaultFontSettings``` to get the default settings.
Also, you can check if we are on electron, you can just use ```platform.os``` to figure our the os.